docs-v1/development/README.adoc

```diff
@@ -19,10 +19,10 @@ If you want to build natively, there are a few prereqs you'll need to go through
 In particular, you'll need the
 https://developers.google.com/protocol-buffers/docs/downloads[protobuf
 compiler].  You'll also need to install the `protoc-gen-gogoslick` toolchain,
-which you can install using `go get`:
+which you can install using `go install`:
 
 ```
-go get -u github.com/gogo/protobuf/protoc-gen-gogoslick
+go install github.com/gogo/protobuf/protoc-gen-gogoslick@latest
```

Comment:
I would leave this file in the previous version because it's for v1 client and that client used the previous version of Go.